SABnzbd.py

```diff
@@ -1086,9 +1086,24 @@ def main():
         if no_file_log:
             logging.info('Console logging only')
 
+    os.chdir(sabnzbd.DIR_PROG)
+
     logging.info('--------------------------------')
+    # If running from git repo, find and give info on branch and commit
+    try:
+        mygitcmd = 'git log -1 --oneline --decorate'.split(' ')
+        gitinfo = sabnzbd.newsunpack.run_simple(mygitcmd).split('\n')[0]
+        if len(gitinfo) >= 5:
+            # We found git info, so store it:
+            sabnzbd.__baseline__ = gitinfo
+            sabnzbd.version.__baseline__ = gitinfo
+    except:
```

Comment:
Might want to change this to `except Exception:` to prevent catching base exceptions (user interrupts and the likes).